lib/LabeledTextItem/LabeledTextItem.less

```diff
@@ -10,7 +10,7 @@
 .moon-labeledtextitem .label {
 	.moon-sub-header-text;
 	color: @moon-label-text-color;
-	border-top: 3px solid @moon-label-border-color;
+	border-top: 0px solid @moon-label-border-color;
```

Comment:
Hi @tejaswineesohoni, the change looks and behaves just fine overall - I think we can probably remove the entire declaration from the rule by taking out this line completely. Thanks!